iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings

```diff
@@ -1866,22 +1866,22 @@
 
 "type.highway.footway" = "Camino";
 
-"type.highway.footway.alpine_hiking" = "Path";
+"type.highway.footway.alpine_hiking" = "Camino";
```

Comment:
The changes in this file seem unrelated.

Reply:
That's ok, translations were not updated properly before.